standard_schema/prerelease/HED8.3.0.mediawiki

```diff
@@ -498,7 +505,32 @@ Each term in this vocabulary has a human-readable description and may include ad
 * Data-property <nowiki>{extensionAllowed}[Something that pertains to data or information.]</nowiki>
 ** Data-artifact <nowiki>[An anomalous, interfering, or distorting signal originating from a source other than the item being studied.]</nowiki>
 *** Biological-artifact <nowiki>[A data artifact arising from a biological entity being measured.]</nowiki>
+**** Eye-artifact <nowiki>[Artifact from eyes.]</nowiki> 
+***** Eye-blink-artifact <nowiki>[Artifact from eye blinking.]</nowiki>
+***** Eye-movement-artifact <nowiki>[Artifact from eye movements.]</nowiki>
+****** Horizontal-eye-movement-horizontal-artifact <nowiki>[Artifact from horizontal eye movements.]</nowiki>
+****** Vertical-eye-movement-artifact <nowiki>[Artifact from vertical eye movements.]</nowiki>
+****** Slow-eye-movement-artifact <nowiki>[Artifact from slow eye movements.]</nowiki>
+****** Nystagmus-artifact <nowiki>[Artifact from nystagmus.]</nowiki>
+**** Chewing-artifact <nowiki>[Artifact from chewing.]</nowiki>
+**** Sucking-artifact <nowiki>[Artifact from sucking.]</nowiki>
+**** Tongue-movement-artifact <nowiki>[Artifact from tongue movement (Glossokinetic).]</nowiki>
+**** Rocking-patting-artifact <nowiki>[Artifact from rocking or patting, most commonly seen in infants.]</nowiki>
+**** Movement-artifact <nowiki>[Artifact from motion.]</nowiki>

```

Comment:
Should we indicate this is motion of the participant?  Is the Rocking-patting a movement or is this due to external influence --- mother patting or rocking an infant?